.github/ISSUE_TEMPLATE/support-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "❓ Support Question"
+about: If you have a question please see our docs or try Slack.
+
+---
+
+If you have a question about using your site, or contributing to the platform, please email support@wordcamp.org or join the `#meta-wordcamp` channel in the official WordPress Slack workspace: https://make.wordpress.org/chat/
```

Comment:
```suggestion
If you have a question about using your site, or contributing to the platform, please email support@wordcamp.org or join the `#meta-wordcamp` channel in [the official WordPress Slack workspace](https://make.wordpress.org/chat/).
```

Same applies to the URLs below.

Reply:
Ideally they'd never see the content post-markdown-conversion though, because this shows up in the issue text editor and the goal is to prevent people from submitting these as issues. So adding `[]()` just adds visual noise to the editor.

Reply:
Ah, that's a great point 👍